packages/mui-docs/package.json

```diff
@@ -36,21 +36,24 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.23.9",
-    "@mui/base": "workspace:*",
-    "@mui/utils": "workspace:^",
     "clsx": "^2.1.0",
     "nprogress": "^0.2.0",
     "prop-types": "^15.8.1"
   },
   "devDependencies": {
+    "@mui/material": "workspace:*",
+    "@mui/icons-material": "workspace:*",
     "@types/node": "^18.19.22",
     "@types/prop-types": "^15.7.11",
     "@types/react": "^18.2.55",
     "next": "^13.5.1",
     "react": "^18.2.0"
   },
   "peerDependencies": {
-    "@mui/material": "workspace:^",
+    "@mui/material": "workspace:*",
+    "@mui/base": "workspace:*",
+    "@mui/icons-material": "workspace:*",
```

Comment:
Can we loosen the `*` requirement? If @mui/docs is released less often than these packages, pnpm will complain as the versions of installed peer dependencies won't match anymore.

Reply:
CSB CI doesn't seem to convert the workspace protocol to actual version numbers (see package.json in https://pkg.csb.dev/mui/material-ui/commit/37466089/@mui/docs).
I don't know about pnpm's `publish`, but CSB itself is a good enough reason to avoid it. Let's use npm-readable version ranges instead (or possibly even `"*"`)

Reply:
🤔  CBS should do it, it does it with the other packages.